fastapi/openapi/docs.py

```diff
@@ -1,45 +1,40 @@
+from typing import Dict
+
 from starlette.responses import HTMLResponse
 
 
-def get_swagger_ui_html(*, openapi_url: str, title: str) -> HTMLResponse:
-    return HTMLResponse(
-        """
+def get_swagger_ui_html(
+    *, openapi_url: str, title: str, swagger_locations: Dict
```

Comment:
Help me change this to accept a `swagger_js_url`, a `swagger_css_url`, and a `swagger_favicon_url`.

All of them `str`, with a default value of the current version.

That way we can document how to re-use these functions.